app/models/incident.rb

```diff
@@ -241,7 +250,7 @@ def claims_fields(table:)
       },
       drivers_report: {
         'FileID' => claims_id,
-      # 'PolicePresent' => report.police_on_scene?,
+        #PolicePresent => report.police_on_scene?,
```

Comment:
while you're here do you need these commented-out fields or are they important notes?

Reply:
From what I can understand the commented out notes are just the fields not being defined when creating objects after the export_to_claims method is called. I'm pretty sure that these can be removed.